internals/cli/cli.go

```diff
@@ -33,7 +33,7 @@ import (
 	"golang.org/x/term"
 
 	"github.com/canonical/pebble/client"
-	"github.com/canonical/pebble/cmd"
+	cmdpkg "github.com/canonical/pebble/cmd"
```

Comment:
Due to clashes with a local variable `cmd` below. Same way it's imported in `cmd_help.go` also.